aries-site/src/pages/components/page.mdx

```diff
@@ -51,8 +51,8 @@ layout styling provided by its parent Page.
 
 <br />
 
-In most cases, all content on a page can be wrapped in a [single PageContent component](#simple).
```

Comment:
I beleive they just have the incorrect link or do we want to remove completely?

Reply:
Seems like the advice to "In most cases, all content on a page can be wrapped in a single PageContent component" is still relevant, but trying to link to a section which has been re-titled or removed?

Reply:
yes that's what I was thinking!